Makefile

```diff
@@ -121,6 +121,12 @@ build-local-container: BIN_SUFFIX=-$(GOOS)
 build-local-container: build ## Builds the provisioner container image using locally built binaries
 	docker build -f Dockerfile.local -t $(IMAGE) .
 
+kind-load-bundles:
+	docker build testdata/bundles/plain-v0/valid -t testdata/bundles/plain-v0:valid
+	docker build testdata/bundles/plain-v0/invalid-crds-and-crs -t testdata/bundles/plain-v0:invalid-crds-and-crs
```

Comment:
#166 added the CONTAINER_RUNTIME variable that defaults to docker. Can we use that instead of hardcoding docker here?

Reply:
Also: should we have a variable that tracks the testdata directory, so changing the repository structure away from a root testdata directory only requires a single change vs. the four references we have here?

Reply:
Also: how does this setup scale if we continue to add additional test Bundle fixtures? Just continue to tack on another line in this kind-load-bundles?

Reply:
Pushed changes to address the first two comments.

For the third comment. I thought about that briefly, but didn't want to get too fancy from the outset. I could imagine a script or a docker-compose sort of task that builds everything and loads it rather than hardcoding everything one by one in the Makefile.